go/engine/scankeys.go

```diff
@@ -314,7 +314,8 @@ func (s *ScanKeys) unlockByID(m libkb.MetaContext, id uint64) openpgp.EntityList
 			Reason:   unlockReason,
 			SecretUI: m.UIs().SecretUI,
 		}
-		unlocked, err := skb.PromptAndUnlock(m, parg, nil, s.me)
+		secretStore := libkb.NewSecretStore(m.G(), s.me.GetNormalizedName())
```

Comment:
why this instead of `m.G().SecretStore()`?

Reply:
SecretStore returns SecretStoreLocked, which is a wrapper for SecretStoreAll. NewSecretStore takes m.G().SecretStore and returns their per-user store.

Reply:
ok